src/main/java/com/hillel/dzhurynskyi/homework/hm20/fabric/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.hillel.dzhurynskyi.homework.hm20.fabric;
+
+public class Main {
+    public static void main(String[] args) {
+        FurnitureFactory factory=new FurnitureFactory();
+
+        factory.furnitureCreate(FurnitureType.CHAIR).create();
```

Comment:
static import can be used to make code more readable: FurnitureType.CHAIR -> CHAIR